bson-adt-core/src/main/scala/adt/bson/BsonValue.scala

```diff
@@ -1,8 +1,8 @@
 package adt.bson
 
-import org.bson.BSON
+import org.bson.BsonType
 import org.bson.types.ObjectId
-import org.joda.time.DateTime
+import org.joda.time.{DateTime, DateTimeZone}
```

Comment:
I would consider it for a more intensive use of DateTime. Here, I am just throwing them into Bson and reading them out again. Nothing fancy.